Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM python:3.11-slim as build
```

Comment:
Does docker-in-docker "just work" here?

Reply:
No, I test Han's code, and myself one, none of them work yet. Need more work on it. In theory, we can get off the Sandbox, as opendevin container can do the same thing. Or, pull the Sandbox out container. Both need more work.

Reply:
You need the Docker CLI installed in the backend container and the socket mounted against it to run docker commands like you would from local. sandbox container will work the same way.

Reply:
Something like this will give you just the CLI:  `apt-get install -y --no-install-recommends docker-ce=5:26.0.0-1~debian.10~`

Reply:
Now that #847 gets merged, the backend can just communicate with the sandbox via `ssh` -- would this simplify things a lot? We could potentially just starts a docker-compose that spins up both sandbox and backend, then have the two container communicate via each other through ssh, instead of docker-in-docker.